.github/workflows/remix_test.yml

```diff
@@ -29,6 +29,19 @@ jobs:
           cache: "npm"
           cache-dependency-path: "remix/package-lock.json"
 
+      - name: Create env file
+        run: |
+          touch .env
+          echo NODE_ENV=development >> .env
+          echo GOOGLE_CLIENT_ID=274093098276-7sar8nsfcpsge9ni7p4nbi5rui6ed3dn.apps.googleusercontent.com >> .env
```

Comment:
I think this is quite dangerous as we are exposing our credentials through our code base 👀, especially while our repo is also a public repo too.